Alexandra/style.css

```diff
@@ -0,0 +1,104 @@
+
+
+body{
+    display: flex;

```

Comment:
try to put the props in alphabetical order. work clean. No extra spaces, use same indentation